tools/rna_tools/graphprot/gplib.py

```diff
@@ -102,7 +102,7 @@ def graphprot_profile_get_top_scores_median(profile_file,
 
 
 ################################################################################
-
+#!/usr/bin/env python3
```

Comment:
Did you mean to remove the existing shebang instead of duplicating it?
The former would make sense for a library with nothing to execute.

Reply:
It's the old one. You want me to remove it?

Reply:
Since there's no main to execute, I could remove it true :)

Reply:
I would remove both this one and the one on top. It's a lib, not a script.

Reply:
Oh, were did this come from? Consider it removed ..

Reply:
I think everything works fine now

Reply:
Thank u very much Wolfgang + Björn

Reply:
Anything else?

Reply:
Tool required memory I would set to 16 GB